pkg/cmd/server/start/start_allinone.go

```diff
@@ -67,14 +67,8 @@ func NewCommandStartAllInOne(basename string, out io.Writer) (*cobra.Command, *A
 		Short: "Launch all-in-one server",
 		Long:  fmt.Sprintf(allInOneLong, basename),
 		Run: func(c *cobra.Command, args []string) {
-			if err := options.Complete(); err != nil {
-				fmt.Fprintln(c.Out(), kcmdutil.UsageError(c, err.Error()))
-				return
-			}
-			if err := options.Validate(args); err != nil {
-				fmt.Fprintln(c.Out(), kcmdutil.UsageError(c, err.Error()))
-				return
-			}
+			kcmdutil.CheckErr(options.Complete())
+			kcmdutil.CheckErr(options.Validate(args))
```

Comment:
General rule is that every component below should return usage errors, I'm ok with returning slightly less info for a time and adding it back.  UsageError right now is almost pointless.